src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -71,58 +68,149 @@ class F extends ValidationTest {
 
 export const g = makeTestGroup(F);
 
-g.test('mipLevelCount_validation_on_formats')
-  .params(poptions('format', kAllTextureFormats))
+g.test('zero_size')
+  .desc(
+    `Test texture creation with zero or nonzero size of
+   width, height, depthOrArrayLayers and mipLevelCount for every dimension, and representative formats.`
+  )
+  .params(
+    params()
+      .combine(poptions('dimension', ['1d', '2d', '3d'] as const))
+      .combine(
+        poptions('zeroArgument', [
+          'none',
+          'width',
+          'height',
+          'depthOrArrayLayers',
+          'mipLevelCount',
+        ] as const)
+      )
+      .combine(
+        poptions('format', [
+          'rgba8unorm',
+          'rgb10a2unorm',
+          'depth24plus-stencil8',
```

Comment:
I'm not even sure whether depth formats can have mipmapping, we need to check this.

Reply:
Removed it and added a TODO. Looks like depth is OK for mipmapping but stencil is not.